crates/infra/cli/src/commands/setup/workspace/mod.rs

```diff
@@ -0,0 +1,18 @@
+use anyhow::Result;
+use infra_utils::{commands::Command, github::GitHub};
+
+pub fn setup_workspace() -> Result<()> {
+    if !GitHub::is_running_in_ci() {
+        warm_up_ide_tools()?;
+    }
+
+    return Ok(());
+}
+
+/// Warm up IDE tools in case it was a fresh installation.
+fn warm_up_ide_tools() -> Result<()> {
+    Command::new("rust-analyzer").flag("--version").run()?;
```

Comment:
I think `rust-analyzer` installs `rust-src` ?

Reply:
I think this is related to the same issue you are seeing in the IDE, where `rust-analyzer` doesn't always install the correct version of `rust-src`. This other call is no-op if it is already installed.